lib/wallaroo/core/source/source_listener.pony

```diff
@@ -25,21 +25,33 @@ use "wallaroo/core/recovery"
 use "wallaroo/core/router_registry"
 use "wallaroo/core/initialization"
 use "wallaroo/core/metrics"
+use "wallaroo/core/network"
 use "wallaroo/core/routing"
 use "wallaroo/core/sink/tcp_sink"
 use "wallaroo/core/source/tcp_source"
 use "wallaroo/core/topology"
 
 interface val SourceListenerBuilder
-  fun apply(env: Env): SourceListener
+  fun apply(env: Env /***SLF, the_journal: SimpleJournal, do_local_file_io: Bool ***/): SourceListener
```

Comment:
What is this comment telling us?

Reply:
Oops, that's cruft that needs to go away.